locales/en.json

```diff
@@ -1611,7 +1611,8 @@
 			"submit": "Submit",
 			"toCourse": "To the course",
 			"tryAgain": "Try again",
-			"yes": "Yes"
+			"yes": "Yes",
+			"loginToSchulcloud": "Login to Schulcloud"
```

Comment:
please dont use "Schulcloud" but the name of the instance. Check in with UX or an experienced frontend developer on which variable to use, I dont know them by heart :)